l10n_it_ricevute_bancarie/i18n/it.po

```diff
@@ -1,39 +1,26 @@
 # Translation of Odoo Server.
```

Comment:
Meglio non modificare i file po* nella PR:
> Important: always translate using Weblate, and never push changes to .po files manually.

(rif. https://odoo-community.org/blog/the-oca-blog-1/post/translations-platform-81)